kubetest/util.go

```diff
@@ -140,8 +141,14 @@ func finishRunning(cmd *exec.Cmd) error {
 
 	finished := make(chan error)
 
+	sig_channel := make(chan os.Signal, 1)
+	signal.Notify(sig_channel, os.Interrupt)
+
 	go func() {
 		finished <- cmd.Wait()
+		sig := <-sig_channel
+		log.Printf("Killing %v after receiving %s", -cmd.Process.Pid, sig)
```

Comment:
Thanks for your review, fejta

OK, I will add it soon.
BTW I created a prototype for this change and I found current way doesn't work.
So I will fix it also on next version.